src/_guides/language/sound-dart.md

```diff
@@ -435,6 +381,11 @@ var arguments = foo['args'];
 The resulting definition depends on the type of `foo` and
 its subscript operator.
 
+{% comment %}
+**[PENDING: That's unclear. Why isn't it `List<String>`?
+  `foo` used to be `methodCall`, if that helps.]**
+{% endcomment %}
+
```

Comment:
Maybe we can just drop this example because it is confusing more than anything else.

> Why isn't it `List<String>`?

All we know is the `foo` yields a value whose type can respond to `operator []` methods and that the type of the argument to `[]` is String.

Reply:
I think it's confusing to use `foo` before it's introduced. Even in the previous example.
I can't see what the example is trying to say, so either figure that out, or remove it entirely.